packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -76,39 +76,46 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
       };
     }
 
-    const GlobalScope = getGlobalScope();
-    if (GlobalScope && GlobalScope.window) {
-      GlobalScope.window.addEventListener('blur', () => {
-        this.stopRecordingEvents && this.stopRecordingEvents();
-        this.stopRecordingEvents = null;
-      });
-      GlobalScope.window.addEventListener('focus', () => {
-        void this.initialize();
-      });
+    const globalScope = getGlobalScope();
```

Comment:
This change was based on looking at the code for page view tracking plugin, to make the reference on globalScope look the same